Manpages for games (#11772)
SerenityOS/serenity: Base/usr/share/man/man6/Hearts.md

## Name

Hearts - The Hearts card game

## Synopsis

```**sh
$ Hearts
```

## Description

> Hearts is a implementation of the 19th century card game.
> 
> The round starts by each player passing three of their cards to a neighbour. After that a number of tricks is played. The player with two of Clubs starts the first trick.
> 
> In each trick the first player chooses freely a card to play (except the first trick where two of Clubs is required) then the other players needs to play a card in the same suit. If a player does not have a card in the same suit they can throw any card. The player who played the highest ranked card in the correct suit wins the trick and gets to start the next trick.

Reviewer: ```suggestion
Hearts is an implementation of the 19th century card game.

The round starts by each player passing three of their cards to a neighbor. Afterwards, a number of tricks is played. The player with the two of clubs starts the first trick.

In each trick, the first player freely chooses a card to play (except the first trick where the two of clubs is required), after which the other players each need to play a card in the same suit. If a player does not have a card in the same suit, they can throw any card. The player who played the highest ranked card in the correct suit wins the trick and gets to start the next trick.
```
